plugins/functions/string.c

```diff
@@ -190,6 +249,10 @@ func_string_substring(grn_ctx *ctx, int n_args, grn_obj **args,
     GRN_TEXT_SET(ctx, substring, start, end - start);
   }
 
+  if(GRN_TEXT_LEN(substring) == 0 && default_value){
```

Comment:
```suggestion
  if (GRN_TEXT_LEN(substring) == 0 && default_value) {
```

It seems that our style and your editor configuration use difference style. How about introducing `.editorconfig` as a separated pull request?

Reply:
Yes, please.